gips/data/smap/smap.py

```diff
@@ -176,13 +176,13 @@ def process(self, *args, **kwargs):
             fname = self.temp_product_filename(sensor, prod_type)  # moved to archive at end of loop
 
             if val[0] == 'smp':
-                img = gippy.GeoImage(allsds[15])
+                img = gippy.GeoImage(allsds[55])
             elif val[0] == 'smpe':
-                img = gippy.GeoImage(allsds[13])
+                img = gippy.GeoImage(allsds[51])
 
             # TODO: This is broken - GeoImage.create() apparently doesn't support datatypes
```

Comment:
looks like you fixed this TODO a couple lines down?

Reply:
:100: percent true.  But otherwise this looks good.  wish this had the "resolve with an issue" feature of gitlab.

Reply:
there's a 'reference in new issue' thing in the ellipsis menu

Reply:
dang!  missed it.  Oh well.  @bhbraswell  I've rebased this branch and am pushing through via gitlab.  So I'm going to close this PR.